pkgs/development/tools/wp4nix/default.nix

```diff
@@ -0,0 +1,30 @@
+{ lib, buildGoModule, fetchFromGitLab, nix, subversion }:
+
+buildGoModule rec {
+  pname = "wp4nix";
+  version = "1.0.0";
+
+  src = fetchFromGitLab {
+    domain = "git.helsinki.tools";
+    owner = "helsinki-systems";
+    repo = "wp4nix";
+    rev = "v${version}";
+    sha256 = "sha256-WJteeFUMr684yZEtUP13MqRjJ1UAeo48AzOPdLEE65w=";
+  };
+
+  vendorSha256 = "sha256-pQpattmS9VmO3ZIQUFn66az8GSmB4IvYhTTCFn6SUmo=";
+
+  postPatch = ''
+    substituteInPlace main.go --replace nix-hash ${nix}/bin/nix-hash
+    substituteInPlace svn.go --replace '"svn"' '"${subversion}/bin/svn"'
+  '';
+
+  meta = with lib; {
+    description = "like wp2nix, but in go and using svn";
```

Comment:
Capitalization.

```suggestion
    description = "Like wp2nix, but in go and using svn";
```

Reply:
This description is generally kind of useless. It does not tell you what this software does. I know it's my description, but that wasn't really intended to be read by anyone.